composer.json

```diff
@@ -26,9 +26,10 @@
         }
     },
     "require": {
-        "symfony/process": "^2.3|^3.0"
+        "symfony/process": "^2.3|^3.0|^4.0"
     },
     "require-dev": {
+        "phpunit/phpunit": "^4.8.35|^5.7",
```

Comment:
:+1: for adding it locally

Reply:
I prefer to require the symfony php unit bridge. Can someone do the PR?

Reply:
phpunit bridge doesn't require phpunit itself.
I see the value of it, yet it's indeed of out scope. for now, we are focusing on making lib usable on php 7+

Reply:
I know, That's why I merged this PR ;)